core/domain/acl_decorators.py

```diff
@@ -39,6 +40,19 @@ def can_play_exploration(handler):
     """Decorator to check whether user can play given exploration."""
 
     def test_can_play(self, exploration_id, **kwargs):
+        """Check whether the user can play the exploration.
+
+        Args:
+            exploration_id: str. The exploration id.
+
+        Returns:
+            bool. Whether the exploration can be
```

Comment:
have you gone through this? (as asked [here](https://github.com/oppia/oppia/pull/4604#discussion-diff-165809829))
I don't think you have gone through the @seanlip's review weel PTAL!

Reply:
@seanlip, Sorry I was too going through this and made the same comment (on the same time) :P. Thanks! for the complete review!
I'm going to delete this.

Reply:
No problem! Thank you for taking a look, too. I was surprised to find it hadn't been addressed yet.